rosidl_typesupport_freertps_cpp/bin/rosidl_typesupport_freertps_cpp

```diff
@@ -42,13 +37,7 @@ def main(argv=sys.argv[1:]):
         return 1
     if rc:
         return rc
-    return generate_dds_freertps_cpp(
-        generator_args['package_name'],
-        generator_args.get('additional_files', []),
-        #args.dds_interface_base_path,
-        generator_args.get('ros_interface_dependencies', []),
-        generator_args['output_dir']
-    )
+    return 0
```

Comment:
You can collapse lines 33-40 into just `return generate_typesupport_freertps_cpp(generator_args)`. The exception will be printed anyway and return non-zero.